### PR TITLE
Tweet eval

### PR DESCRIPTION
**Pull request by arunraja-hub**

### CODE REVIEW

**Review by VictorSanh — CHANGES_REQUESTED** [review]
Thank you @arunraja-hub ! 

Could you add variants for where you specify the options in the prompt for `emoji`, `emotion`, `sentiment`, `stance_*`, etc.)?
Yes/Nos are fine I think, but the others have quite a lot of options and the model will end up just learning the labels (so let's give them to the model)

I think it's worth it to write a few more templates per subset. `yelp_polarity`'s templates are a good example of creativity and are quite close to this dataset (`hate`, `irony`, `offensive`). Please also protect the labels (for instance `{{"hate"}}`)

File: templates/tweet_eval/emoji/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+subset: emoji
+templates:
+  8c794abe-5364-430f-aa1e-eb3501443cec: !Template
+    id: 8c794abe-5364-430f-aa1e-eb3501443cec
+    jinja: "{% set emo = [\n  \"\u2764\",\n  \"\U0001F60D\",\n  \"\U0001F602\",\n\
+      \  \"\U0001F495\",\n  \"\U0001F525\",\n  \"\U0001F60A\",\n  \"\U0001F60E\",\n\
+      \  \"\u2728\",\n  \"\U0001F499\",\n  \"\U0001F618\",\n  \"\U0001F4F7\",\n  \"\
+      \U0001F1FA\U0001F1F8\",\n  \"\u2600\",\n  \"\U0001F49C\",\n  \"\U0001F609\"\
+      ,\n  \"\U0001F4AF\",\n  \"\U0001F601\",\n  \"\U0001F384\",\n  \"\U0001F4F8\"\
+      ,\n  \"\U0001F61C\"] %}\n\nWhich emoji among {{emo}} best describes the sentiment\
```
Comment (VictorSanh) [inline templates/tweet_eval/emoji/templates.yaml:11]: {{emo}} is not compiling, you need to join the list: {{emo | join(", ")}}.
same for `li` and subset sentiment.

Comment (arunraja-hub) [reply templates/tweet_eval/emoji/templates.yaml:11]: On my side emo does compile.
Anyway I have added joined emo now

**Review by VictorSanh — APPROVED** [review]

**Comment by VictorSanh** [comment]
merging now, please update the spreadsheet!

**Pull request merged**